DOIP-story.md

```diff
@@ -216,13 +216,15 @@ Sasha begins to fade away into the abyss, and Minogar rushes in - his blood boil
 
 Marlade fires his crossbow from distance, and launches an arrow like a bullet straight into the eye of another Goblin, killing it instantly.
 
-Sensing her time is short, Marlade drags Sasha back to the group and Marlade plants his shield in the dirt between her and the remaining Goblin.
+Sensing her time is short, Minogar drags Sasha back to the group and Marlade plants his shield in the dirt between her and the remaining Goblin. 
+
+Lanar rushe's in seeing Sasha's condition to stabalise her.
```

Comment:
Shouldn't this just be `rushes` ?

Reply:
Yeah you're right, I had Rosie jumping up on me like the a-hole she is 😂

Reply:
That should be correct now @jflanaganuk